Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/webapp/WebAppNode.java

```diff
@@ -91,15 +84,21 @@ public void renderSubModules() {
 
     @Override
     protected void loadActions() {
```

Comment:
configureBasicActionBuilder -> buildBasicAction

Reply:
It's not action. it is a builder. so I rename it to initActionBuilder